translations/Hindi/special-terms.md

```diff
@@ -0,0 +1,107 @@
+ खोलें
+* Private/Invite-Only Stream : निजी / केवल आमंत्रण स्ट्रीम , Literal 
+translation is used here, therefore there will be no confusion on any side
+ while this word is used. , अपने निजी / केवल आमंत्रण स्ट्रीम देखें
+* Public Stream : सार्वजनिक स्ट्रीम , सार्वजनिक literally means "For all 
+people", and public streams are also for all users. Therefore this translation
+ is used. Other translations which are rejected are: प्रसिद्ध(famous), 
+ सामान्य(common), and प्रचलित(prevalent) , सार्वजनिक स्ट्रीम देखें
+* Bot : बॉट , Transliteration used here. Translation is available for "Robot"
+ instead, but that translation literally means a humanoid, whereas bots in
+ Zulip are just programs. , अपने बॉट देखें
+* Integration : एकीकरण , एकीकरण to any average Hindi user would mean something
+ like unification, and since it is used in Zulip, it would mean unifying 
+ Zulip with something. , फेसबुक के साथ जुलिप एकीकरण
+* Notification : अधिसूचना , अधिसूचना is the easiest to understand translation 
+of "Notification" and it literally means "Before information/Pre-information",
+ which is basically what notifications are: Alerting the user about a piece of
+ information waiting for them. , अधिसूचनाएं बंद करो
+* Alert Word : सतर्क शब्द , चेतावनी,सावधान,भय which are translations of alert 
+are all related to being scared and as a result alert. Therefore सतर्क is used
+ as a translation of alert. , अपने सतर्क शब्द देखें
+* View : देखें , One translation of view is राय, but that means opinion. 
+Therefore the formal translation of see is used here , बिना पढ़े हुए पोस्ट देखें
+* Home : मुख्य पृष्ठ , Translation of homepage is used here with a slight 
+modification. मुख पृष्ठ(home page) literally means face page or front page,
+ usually associated with books and magazines. Therefore मुख्य पृष्ठ is used
+ which means "Main page" , मुख्य पृष्ठ पर जाएं
+* Emoji : इमोजी , Transliteration used here, as there are no possible 
+translations for "emoji". Also the common Hindi folk is familiar with the 
+English word and therefore using transliteration shouldn't be a problem , इमोजी
+ शैली बदलें
+## Phrases (वाक्यांश)
+* Subscribe to a stream/Unsubscribe from a stream : किसी स्ट्रीम की सदस्यता लें
+ / एक स्ट्रीम से सदस्यता समाप्त करें , There is no direct word for "Subscribe"
+ in Hindi, but the translator result means "Take a subscription". Similarly the
+ result of "Unsubscribe" means "End a subscription". Both of these translations
+ are used here , <stream> की सदस्यता लें / सदस्यता रद्द करें
+* Narrow to : द्वारा संकुचित करें , The translation of "Narrow by" is used 
+instead of "Narrow to", which when translated means "physically narrow an 
+object to something", which cannot be used in this context , इस विषय द्वारा 
+संकुचित करें
+* Filter : फ़िल्टर , One available translation of filter is छन्ना, but it 
+literally means a strainer. Therefore translitration is used here. , फ़िल्टर 
+हटाए
+* Mute/Unmute : म्यूट / अनम्यूट , Transliteration is used here as welll as 

```

Comment:
Mute can be *shaant* as well (translation for silent). However, in that case *unmute* would be a challenge.

Reply:
That was the reason I had to go with transliteration as अशांत is usually associated with noise and doesn't sound appropriate to use in Zulip. Also if we use those words, it could mean "pacify the chaos in the stream"